sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -272,6 +272,14 @@ int _icmpv6_ping(int argc, char **argv)
             puts("ping timeout");
         }
 
+        while(msg_try_receive(&msg) > 0) {
```

Comment:
I guess so. This loop is for handling duplicates that arrive during the delay period, the other loop is for handling duplicates that potentially arrive after the unregistering.